aio_pika/robust_channel.py

```diff
@@ -67,7 +67,7 @@ async def restore(self) -> None:
         await self.default_exchange.restore(self)
 
         for exchanges in self._exchanges.values():
-            for exchange in exchanges:
+            for exchange in exchanges.copy():
```

Comment:
This narrows down the scope of refactoring a bit in the future. In my opinion, it would be enough to do `tuple(exchanges)` here. `tuple()` requires something iterable as input and can not modifying.